hal/armv7a/zynq7000/timer.c

```diff
@@ -52,14 +55,14 @@ static int _timer_irqHandler(unsigned int n, cpu_context_t *ctx, void *arg)
 	spinlock_ctx_t sc;
 	hal_spinlockSet(&timer_common.sp, &sc);
 	/* Interval IRQ */
-	if ((*(timer_common.ttc + isr) & 1) != 0) {
-		timer_common.jiffies += timer_common.ticksPerFreq;
+	if ((*(timer_common.ttc + isr) & 1U) != 0U) {
+		timer_common.jiffies += (time_t)timer_common.ticksPerFreq;
 	}
 
 	hal_spinlockClear(&timer_common.sp, &sc);
 
-	u32 nextID = hal_cpuGetID() + 1;
-	u32 nextTargetCPU = (nextID == hal_cpuGetCount()) ? 1 : (1 << nextID);
+	u32 nextID = hal_cpuGetID() + 1U;
+	u32 nextTargetCPU = (nextID == hal_cpuGetCount()) ? 1U : (1UL << nextID);
```

Comment:
`1U` is the same width as `1UL` on this platform, so `1U` should be fine.
```suggestion
	u32 nextTargetCPU = (nextID == hal_cpuGetCount()) ? 1U : (1U << nextID);
```